src/puppetlabs/puppetdb/middleware.clj

```diff
@@ -50,12 +51,9 @@
       (if (allowed? req)
         :authorized
         (do
-          (log/warnf "%s rejected by certificate whitelist %s" ssl-client-cn whitelist)
-          (format (str "The client certificate name (%s) doesn't "
-                       "appear in the certificate whitelist. Is your "
-                       "master's (or other PuppetDB client's) certname "
-                       "listed in PuppetDB's certificate-whitelist file?")
-                  ssl-client-cn))))))
+          (log/warnf (i18n/trs "{0} rejected by certificate whitelist {1}" ssl-client-cn whitelist))
+          (i18n/tru "The client certificate name {0} doesn''t appear in the certificate whitelist. Is your master''s (or other PuppetDB client''s) certname listed in PuppetDB''s certificate-whitelist file?"
```

Comment:
Just for clarification, `xgettext --lang=lisp` doesn't handle `(str` wraps or line breaks, so we're probably stuck with this.